### PR TITLE
Feature owen eth call demo

### PR DESCRIPTION
**Pull request by yubing744**
I add a web project to the counter contract and complete the coding of the contract call

image

Ref https://github.com/rooch-network/rooch/issues/278

### CODE REVIEW

File: examples/counter/web/src/lib/rooch_types.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
```
Comment (jolestar) [inline examples/counter/web/src/lib/rooch_types.yml:1]: We can consider auto-generate rooch_types.yml in the ci.

**Review by jolestar — APPROVED** [review]
LGTM, @wow-sven take a look.

**Review by wow-sven — APPROVED** [review]

File: examples/counter/web/src/lib/rooch_types.yml (hunk 1 above)
Comment (yubing744) [reply examples/counter/web/src/lib/rooch_types.yml:1]: I have tried to use serde-generate to generate TS, but an UnknownFormatInContainer error will appear. Through exclusion, it is determined that the indirect recursive definition of pub type_params: TypeTag in the StructTag structure causes serde_generate to report an error.

Comment (yubing744) [reply examples/counter/web/src/lib/rooch_types.yml:1]: If we use serde-generate to generate rooch_types.yaml should report the same error. Later, we need to fix the bug of serde-generate before we can generate rooch_types.yaml through CI.

**Comment by yubing744** [comment]
Now there are a lot of MoveAction coding codes in this web Demo, and it feels like it needs to be refactored into rooch.js.

File: examples/counter/web/src/lib/rooch_types.yml (hunk 1 above)
Comment (jolestar) [reply examples/counter/web/src/lib/rooch_types.yml:1]: help to record an issue.

Comment (yubing744) [reply examples/counter/web/src/lib/rooch_types.yml:1]: > help to record an issue.

OK

**Comment by yubing744** [comment]
I added two subtasks:
https://github.com/rooch-network/rooch/issues/305
https://github.com/rooch-network/rooch/issues/306

**Pull request merged**